### DISCUSSION TITLE
Default value for @Question

### DISCUSSION THREAD

**Discussion by simplenotezy** [jmcdo29/nest-commander#827] [Q&A]
Is there a way to provide default values for the `@Question`?

**Comment by jmcdo29** [comment]
You could do something like
```ts
@QuestionSet({ name: 'task-questions' })
export class TaskQuestions {
  @Question({
    message: 'What task would you like to execute?',
    name: 'task'
  })
  parseTask(val?: string) {
    return val ?? 'dedfault value';
  }
}
```

or you could make use of the [`@DefaultFor()`](https://nest-commander.jaymcdoniel.dev/en/features/inquirer/#inquirer-decorators) decorator and method
